api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -75,6 +75,9 @@
 	
 	public static final String OPENMRS_FHIR_EXT_MEDICINE = OPENMRS_FHIR_EXT_PREFIX + "/medicine";
 	
+	public static final String OPENMRS_FHIR_EXT_RELATIONSHIP_IDENTIFIER = OPENMRS_FHIR_EXT_PREFIX
+	        + "/Relationship/identifier";
+	
```

Comment:
Why are we adding this new extension here?

Reply:
I am adding an identifier to refer to person-A.